.markdownlintrc

```diff
@@ -0,0 +1,6 @@
+{
+    // Disable line length check
+    "MD013": false,
+    // Disable inline HTML check
+    "MD033": false
+} 
```

Comment:
any comments/reasons on these specific settings?

Reply:
Good call, I'll add them in. 

`line length` => so we can just write normally (unless you think it's preferable to break up paragraphs and write line-by-line)

`inline HTML` => allows us to manually use duplicate headings with separate `id`s and point to the correct heading based on the `id`